tdrs-backend/tdpservice/email/email.py

```diff
@@ -0,0 +1,90 @@
+"""
+Send emails.
+
+This module currently uses the EmailMultiAlternatives class from Django to send emails with HTML content.
+A optional plain text message can be included as well. Emails should be sent using mail.delay()
+
+The mail function is the main entry point for sending emails.
+It takes in a EmailType enum(that has a template path as the value), a list of recipients,
+and a dictionary of context variables to be used in the email template. The context vlaues can be found in the
+templates directory.
+
+There are optimizations that can be made to this module for sending mass emails.
+Django's send_mass_mail function can be used to send multiple emails at once.
+https://docs.djangoproject.com/en/4.1/topics/email/#send-mass-mail
+"""
```

Comment:
Please rewrite the first line and/or simplify generally. Additionally, let's please have imports first in the file.